games-fps/prboom-plus/prboom-plus-2.5.1.4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools desktop xdg
+
+DESCRIPTION="A Doom source port developed from the original PrBoom project"
+HOMEPAGE="http://prboom-plus.sourceforge.net"
+# We are using a github mirror here because the upstream tarball is missing the
+# free dog assets; we should update SRC_URI to point to the upstream tarball
+# once the free assets are present as expected
+#SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/coelckers/prboom-plus/archive/f96f891d068dcc5ec52ed91056b46d27e9a8462d.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+
+	nonfree? ( freedist )
+	!nonfree? ( BSD )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dumb fluidsynth +mad net nonfree +opengl pcre +png portmidi sdl-image +sdl-mixer +vorbis"
+REQUIRED_USE="sdl-image? ( opengl )"
+
+DEPEND="
+	media-libs/libsdl
+	dumb? ( media-libs/dumb )
+	fluidsynth? ( media-sound/fluidsynth )
+	mad? ( media-libs/libmad )
+	net? ( media-libs/sdl-net )
+	opengl? (
+		virtual/opengl
+		virtual/glu
+		sdl-image? ( media-libs/sdl-image )
+	)
+	pcre? ( dev-libs/libpcre )
+	png? ( media-libs/libpng:= )
+	portmidi? ( media-libs/portmidi )
+	sdl-mixer? ( media-libs/sdl-mixer[midi,timidity] )
```

Comment:
Please don't add `[timidity]` to sdl-mixer dependencies. I poured hours into implementing FluidSynth support so that users would not have to suffer the awfulness that is Timidity. The package ensures you have at least one of them when `midi` is enabled.